converters/google/resources/cloudrun_domain_mapping.go

```diff
@@ -62,8 +62,8 @@ func GetCloudRunDomainMappingCaiObject(d TerraformResourceData, config *Config)
 			Name: name,
 			Type: CloudRunDomainMappingAssetType,
 			Resource: &AssetResource{
-				Version:              "v1",
-				DiscoveryDocumentURI: "https://www.googleapis.com/discovery/v1/apis/{{location}}-run/v1/rest",
+				Version:              "run.googleapis.com",
```

Comment:
looking at other assets, it seems like this should be the version (i.e. v1, v2, etc) rather than the domain